ShiftPlan.Blazor.Client.Tests/LocalFileServiceTests.cs

```diff
@@ -0,0 +1,43 @@
+using FluentAssertions;
+using ShiftPlan.Blazor.Client.Services;
+using ShiftPlan.Blazor.Commons.Models;
+using ShiftPlan.Blazor.Commons.Services;
+
+namespace ShiftPlan.Blazor.Client.Tests
+{
+    public class LocalFileServiceTests
+    {
+        private string jsonFile = Path.Combine( Directory.GetParent(AppDomain.CurrentDomain.BaseDirectory).FullName,"Employee.json");
+        private ILoadSaveService<Employee> _sut { get; }
```

Comment:
nope. `_sut` should be instance of class what you are testing. 
```suggestion
        private LocalFileService<Employee> _sut { get; init; }
```